chromatic/rainbows/readers/x1dints.py

```diff
@@ -178,6 +199,9 @@ def from_x1dints(rainbow, filepath, order=None, **kw):
     # create a list of filenames (which might be just 1)
     filenames = expand_filenames(filepath)
 
+    # extract the versions of the calibration pipeline
```

Comment:
This worked on the old x1dints files I had on my computer; haven't tested on the newer ones but I trust you it helps!